server/package.json

```diff
@@ -18,7 +18,7 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "test:e2e": "./scripts/run-integration.sh",
```

Comment:
I think this run-integration using docker-compose should only be for CI
for devs, we should run against a pre-provisionned "test" database. We only need to use the right .env and the server should directly connect to it